docs/pages/kubernetes-access/helm/guides/custom.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Running a Teleport cluster with a custom configuration using Helm
+title: Running Teleport with a Custom Configuration using Helm
```

Comment:
custom configuration is a proper noun?

Reply:
We're using title case for guide titles and sentence case for subheadings (https://goteleport.com/docs/contributing/documentation/style-guide/#page-titles).